File: JS, PHP, PYTHON/PYTHON/DICCIONARIO.py
diccionario = {'Euro':'€', 'Dollar':'$', 'Yen':'¥', 'Peso':'$'}

peso = diccionario

def conversor():
    divisa = input("Ingrese el tipo de divisa: ").capitalize()  
    if divisa in peso:
        print("Esta es la divisa:", peso[divisa])
    else:
        print("Divisa no encontrada.")
        
conversor()

File: JS, PHP, PYTHON/PHP/DICCIONARIO.php
<?php
$diccionario = array(
    'Euro' => '€',
    'Dollar' => '$',
    'Yen' => '¥',
    'Peso' => '$'
);

function conversor($diccionario) {
    $divisa = ucfirst(trim(readline("Ingrese el tipo de divisa: ")));
    
    if (array_key_exists($divisa, $diccionario)) {
        echo "Esta es la divisa: " . $diccionario[$divisa] . "\n";
    } else {
        echo "Divisa no encontrada.\n";
    }
}

conversor($diccionario);
?>